src/renderer/services/bitcoin/transaction.ts

```diff
@@ -30,7 +30,7 @@ export const createTransactionService = (client$: Client$, network$: Network$):
       amount,
       sender,
       feeRate,
-      feeOption: undefined,
+      feeOption: feeOption,
```

Comment:
We don't need to assign the value here.

Just `feeOption,` would be okay.

Reply:
good catch